david_vallejos/space_challenge/src/main/java/com/tech_group/Main.java

```diff
@@ -0,0 +1,40 @@
+package main.java.com.tech_group;
+
+import java.text.NumberFormat;
+import java.util.ArrayList;
+
+/**
+ * Class to start simulations.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+
+        final String filePathPhase1 = "F:\\SDF\\TechGroup1-20\\david_vallejos\\space_challenge\\src\\Phase-1.txt";
+        final String filePathPhase2 = "F:\\SDF\\TechGroup1-20\\david_vallejos\\space_challenge\\src\\Phase-2.txt";
```

Comment:
If a download the code and I try to run it I'm sure that I'll get an error because there doesn't exist an F:\\SDF directory on my pc, please call your files from your project.

Reply:
Done, fixed